### PR TITLE
docs: Add in draft testing plan

### PR DESCRIPTION
**Pull request by jhesketh**
This is from our discussion:
https://github.com/aquarist-labs/forum/discussions/16

This is just the initial doc, improvements and expansion should be done as followup commits.

### CODE REVIEW

File: doc/project-plan/testing-plan.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+-------
+Jenkins is configured to run `wasser` on each PR. The jobs are defined in 
+https://github.com/aquarist-labs/wasser/blob/main/examples/jjb/aquarium.yaml
+
+aqrtest
+-------
+
+This is not run as part of any CI yet.
+
+TODO
```
Comment (Devp00l) [inline doc/project-plan/testing-plan.rst:133]: Is this TODO mandatory or should that replace the issue that should be created to run aqrtest in CI?

**Review by Devp00l — COMMENTED** [review]
Sounds great so far :)

File: doc/project-plan/testing-plan.rst (hunk 1 above)
Comment (jhesketh) [reply doc/project-plan/testing-plan.rst:133]: The TODO here is to document what aqrtest does and how to use it

**Review by asettle — APPROVED** [review]
LGTM.

**Pull request merged**